src/store/modules/ResourceManagement/SystemParametersStore.js

```diff
@@ -71,6 +250,121 @@ const systemParametersStore = {
+      };
+      return await api
+        .patch(
+          '/redfish/v1/Systems/system/Bios/Settings',
+          updatedRpdFeatureValue
+        )
+        .then(() => {
+          commit(
+            'setRpdFeature',
+            updatedRpdFeatureValue.Attributes.pvm_rpd_feature
+          );
+          return i18n.t('pageSystemParameters.toast.successSavingRpdFeature');
+        })
+        .catch((error) => {
+          console.log('error', error);
+          throw new Error(
+            i18n.t('pageSystemParameters.toast.errorSavingRpdFeature')
+          );
+        });
+    },
+    async saveImmediateTestRequested({ commit }) {
+      commit('setImmediateTestRequested', true);
+      const updatedImmediateTestRequestedValue = {
+        Attributes: { pvm_rpd_immediate_test: 'Enabled' },
+      };
+      return api
+        .patch(
+          '/redfish/v1/Systems/system/Bios/Settings',
+          updatedImmediateTestRequestedValue
+        )
+        .then(() => {
+          return i18n.t(
+            'pageSystemParameters.toast.successSavingImmediateTestRequested'
+          );
+        })
+        .catch((error) => {
+          console.log(error);
+          commit('setImmediateTestRequested', false);
+          throw new Error(
+            i18n.t(
+              'pageSystemParameters.toast.errorSavingImmediateTestRequested'
+            )
+          );
+        });
+    },
+    async saveGardOnError({ commit }, updatedImmediateTestRequested) {
+      let updatedValue = updatedImmediateTestRequested ? 'Enabled' : 'Disabled';
+      commit('setGardOnError', updatedImmediateTestRequested);
+      const updatedImmediateTestRequestedValue = {
+        Attributes: { pvm_rpd_gard_policy: updatedValue },
+      };
+      return api
+        .patch(
+          '/redfish/v1/Systems/system/Bios/Settings',
+          updatedImmediateTestRequestedValue
+        )
+        .then(() => {
+          return i18n.t('pageSystemParameters.toast.successSavingGardOnError');
+        })
+        .catch((error) => {
+          console.log(error);
+          commit('setGardOnError', !updatedImmediateTestRequested);
+          throw new Error(
+            i18n.t('pageSystemParameters.toast.errorSavingGardOnError')
+          );
+        });
+    },
+    async saveRpdScheduledRun({ commit }, payload) {
+      const updatedIoEnlargedCapacity = {
+        Attributes: {
+          pvm_rpd_scheduled_tod: payload.totalSeconds,
+          pvm_rpd_scheduled_duration: payload.duration,
+        },
+      };
+      return await api
+        .patch(
+          '/redfish/v1/Systems/system/Bios/Settings',
+          updatedIoEnlargedCapacity
+        )
+        .then(() => {
+          commit('setRpdScheduledRun', payload.startTime);
+          return i18n.t('pageMemory.toast.success');
+        })
+        .catch((error) => {
+          console.log('error', error);
+          throw new Error(i18n.t('pageMemory.toast.error'));
```

Comment:
Similarly from pageMemory.toast.error: **error** doesn't exist in the translation file